tools/pot/openvino/tools/pot/api/samples/classification/classification_simple_api_sample.py

```diff
@@ -0,0 +1,62 @@
+# Copyright (C) 2020-2022 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+from inspect import Parameter
+import os
+from sqlite3.dbapi2 import _Parameters
+import numpy as np
+from cv2 import imread, resize as cv2_resize
+
+from openvino.tools.pot import quantize_post_training, QuantizationParameters, export
+from openvino.tools.pot.api.samples.utils.argument_parser import get_common_argparser
+
+from .classification_sample import ImageNetDataLoader
+
+def main():
+    argparser = get_common_argparser()
+    argparser.add_argument(
+        '-a',
+        '--annotation-file',
+        help='File with Imagenet annotations in .txt format',
+        required=True
+    )
+
+    args = argparser.parse_args()
+
+    if not args.weights:
+        args.weights = '{}.bin'.format(os.path.splitext(args.model)[0])
+
+    dataset_config = {
+        'data_source': os.path.expanduser(args.dataset),
+        'annotation_file': os.path.expanduser(args.annotation_file),
+        'has_background': True,
+        'preprocessing': [
+            {
+                'type': 'crop',
+                'central_fraction': 0.875
+            },
+            {
+                'type': 'resize',
+                'width': 224,
+                'height': 224
+            }
+        ],
+    }
+
+    # Step 1: Create a user-defined DataLoader object
+    data_loader = ImageNetDataLoader(dataset_config)
```

Comment:
It would be nice to have DataLoaderParameters similarly to QuantizationParameters

Reply:
This is temporary stuff. We are planning to use framework implementations. No configs in the future, it is supposed to be a Python code.